src/main/resources/migration/common/V32__support_storing_case_admins.sql

```diff
@@ -0,0 +1,25 @@
+ALTER TABLE community_offender_manager ADD COLUMN kind varchar(30);
+UPDATE community_offender_manager SET kind = 'COMMUNITY_OFFENDER_MANAGER';
+ALTER TABLE community_offender_manager ALTER COLUMN kind SET NOT NULL;
+
+ALTER TABLE community_offender_manager ALTER COLUMN staff_identifier DROP NOT NULL;
```

Comment:
Is this removing the staff_identifiers present in the table?

Reply:
no - it drops the not null constraint
It definitely sounds like it could delete data so I double checked!